src/DTS/Repository/TemplateRepository.php

```diff
@@ -145,7 +141,30 @@ public function filterByType($type) {
   public function filterBySearchString($searchString) {
     $cols = ['templateId','docType','templateKey','name','author','createdAt'];
     $rows = $this->persistence->filterBySearchString($searchString, $cols);
-    return array_map(function($row) {
+    return $this->toTemplateArray($rows);
+  }
+
+  /**
+   * @param string $type
+   * @param string $key
+   * @return Template[]
+   */
+  public function getVersions(string $type, string $key) {
```

Comment:
Appears to returns a list sorted by bodyUri, ascending.  Since this function returns a "history", consider changing the sort order where most recently modified items are listed first (createdAt, descending).